android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/theme/Shape.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.ic.kmm.suv.android.ui.theme
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Shapes
+import androidx.compose.ui.unit.dp
+
+val Shapes = Shapes(
+    small = RoundedCornerShape(4.dp),
+    medium = RoundedCornerShape(4.dp),
+    large = RoundedCornerShape(0.dp)
```

Comment:
Small == Medium? 🤔

Reply:
@hoangnguyen92dn Yes! Just followed the Shapes class from `androidx.compose.material`

<img width="818" alt="Screen Shot 2022-09-21 at 12 08 35" src="https://user-images.githubusercontent.com/19943832/191419034-08bc2d74-3671-4893-8d17-0644ab056145.png">

I will update the value later when we use it.

Reply:
It's confused a bit for new comers IMO 😄 I'd suggest to remove it if it's not being in use. It's up to you ✋

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/79/commits/f734fc016f53c04d98972caa71f5415472473153